src/main/java/io/lighty/yang/validator/formats/JsonTree.java

```diff
@@ -90,12 +90,13 @@
     private static final String SLASH = "/";
     private static final String COLON = ":";
 
-    private final Map<String, String> prefixMap = new HashMap<>();
+    private Map<String, String> prefixMap = new HashMap<>();
```

Comment:
it looks like it can be changed to local variable

Reply:
hm... values are put int map only, no no getting vealues from map, so it looks like the variable is useless here

Reply:
hm... values are put int map only, no no getting vealues from map, so it looks like the variable is useless here

Reply:
done